static/Events/PPWorkshop.html

```diff
@@ -0,0 +1,159 @@
+            <div id="navBar">
+                <div id="navBarLogoContainer">
+                    <a href="../index.html" title="MCC Homepage"></a>
+                </div>
+                <div id="navBarCenterSpacing"></div>
+                <nav id="navMenu">
+                    <span><a href="../mission.html">Our Mission</a></span>
+                    <span><a href="../team.html">The Team</a></span>
+                    <span><a href="../events.html">Events</a></span>
+                    <span><a href="../join.html" class="mccRed">Join Now</a></span>
+                </nav>
+                <div class="menuIconContainer">
+                    <a href="#navMenu">
+                        <div class="bar1"></div>
+                        <div class="bar2"></div>
+                        <div class="bar3"></div>
+                    </a>
+                </div>    
+            </div>
+        </header>
+
+        <main id="contentContainer">
+
+            <section id="introSection">
+                <div class="sectionDiv">
+                    <div id="introText">
+                        <p id="pageTitle" class="mccBlue">22.01.2021 - ONLINE</p>
+                        <h1>Power Platform Workshop</h1>
+                    </div>
+                </div>
+            </section>
+
+            <section id="IntroSection">
+                <div class="sectionDiv">
+                    <img id="eventIntroImg" class="boxShadow" src="../Assets/Images/power-platform-icons.jpg" alt="A placeholder image">
+                </div>
+            </section>
+
+            <section id="DescriptionSection">
+                <div id="Description" class="sectionDiv">
+                    <div id="overview">
+                        <h4>Date</h4>
+                        <p>22.01.2021</p>
+                        <br/>
+                        <h4>Time</h4>
+                        <p>04:00 pm - 05:00 pm CEST</p>
+                        <br/>
+                        <h4>Place</h4>
+                        <p>Teams Online Meeting</p>
+                        <p><a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_ODIzNDlhZmQtMTY2Mi00NThlLWJhYzctODk1MzU5YTMwMmI0%40thread.v2/0?context=%7b%22Tid%22%3a%2284c31ca0-ac3b-4eae-ad11-519d80233e6f%22%2c%22Oid%22%3a%227c85ffcf-b716-4202-b5e0-22df34d099f4%22%7d" style="color: var(--color-mcc-blue)">Click here to join</a></p>
+                    </div>
+                    <div id="endText">
+                        <h2>Low-code / no-code development for all!</h2>
+                        <p>Lee Zuckett, Program Manager for the Power Platform, will teach us how to build no-code / low-code solutions.</p>
+                        <h2>The next wave in software development has nothing to do with coding.</h2>
+                        <p>
+                            According to Microsoft, 500 million new apps will be built in the next 5 years. This is more than the number of apps built in the last 40 years. 
```

Comment:
According to Microsoft, 500 million new apps will be built in the next 5 years. This is **even** more than the number of apps built in the last 40 years.